tastytrade/instruments.py

```diff
@@ -91,6 +90,26 @@ def get_cryptocurrency(cls, session: Session, symbol: str) -> 'Cryptocurrency':
         return cls.from_dict(data)
 
 
+@dataclass
+class TickSizes:
```

Comment:
I don't think this class is providing much value, maybe it's better to just return the raw JSON dicts?

Reply:
Actually, we could go a step further and get rid of the dataclasses that don't have additional functionality (DestinationVenueSymbol, Pagination); this will make the code a bit cleaner and faster, though as a downside we'd no longer need recursive snakes :(

Reply:
I wouldn't touch TickSizes, it might be helpful in special cases, but the unneeded classes can be removed, I agree.

Reply:
To be clear removing the class doesn't mean the data isn't there; it just means it's perhaps slightly less clear that it's there, but in exchange it's less code and faster.

Reply:
Hmm.. I was wondering why would we remove DestinationVenueSymbol, seems useful. (i removed the pagination dataclass)

Reply:
Well, if you think about it the only difference between having it, assuming `crypto` is an instance of `Cryptocurrency` and not is:
`crypto.destination_venue_symbols[0].destination_venue`
vs
`crypto.destination_venue_symbols[0]['destination_venue']`

I think it is useful; however, it should probably be a `TypedDict` instead of a separate dataclass, given it has no functionality apart from representing data.